docs/_includes/generated/rocks_db_configurable_configuration.html

```diff
@@ -56,6 +56,12 @@
             <td>String</td>
             <td>The maximum number of concurrent background flush and compaction jobs (per TaskManager). RocksDB has default configuration as '1'.</td>
         </tr>
+        <tr>
+            <td><h5>state.backend.rocksdb.write-batch-size</h5></td>
```

Comment:
Use the config with the new type `MemorySize`(introduced by [FLINK-14491](https://issues.apache.org/jira/browse/FLINK-14491))